corporate/tests/test_stripe.py

```diff
@@ -789,10 +789,21 @@ def test_billing_not_enabled(self) -> None:
             self.assertEqual(response.status_code, 404)
 
     @mock_stripe()
-    def test_get_past_invoices_session_url(self, *mocks: Mock) -> None:
+    def test_stripe_billing_portal_urls(self, *mocks: Mock) -> None:
```

Comment:
A new test would have been almost identical to those for the invoices billing portal, so I combined them so that we wouldn't have another 100 stripe test fixtures.